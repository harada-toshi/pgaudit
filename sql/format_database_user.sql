\! cp sql/format_database_user/format_database_user.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log

\! pg_ctl -w restart -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! sh sql/format_database_user/format_database_user.sh

\! rm /tmp/config.conf

\! grep "AUDIT:" ./tmp_check/tmp_check.log > results/format_database_user.out

