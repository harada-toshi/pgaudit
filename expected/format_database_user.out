	        format = 'AUDIT:(1) %database,%user,%current_user,%command_text'
	        format = 'AUDIT:(2) %d,%u,%current_user,%command_text'
LOG:  AUDIT:(1) foo,foo_user,foo_user,SELECT 1
LOG:  AUDIT:(2) foo,foo_user,foo_user,SELECT 1
